documentation/modules/auxiliary/scanner/portscan/tcp.md

```diff
@@ -0,0 +1,69 @@
+## Description
+  
+   This module will Enumerate open TCP services by performing a full TCP     connect on each port. This will establish a complete three-way handshake (SYN -> SYN/ACK -> ACK) on the target port. This does not need administrative privileges on the sourcemachine, which may be useful if pivoting.
+
+## Vulnerable Application
+
+  Any reachable TCP endpoint is a potential target.
+
+## Options
+
+  **PORTS**
+  This is the list of ports to test for TCP Scan on each host.
+  Formats like  `1-3`, `1,2,3`, `1,2-3`, etc. are all supported.Default
+  options is to scan `1-10000` ports.
+ 
+  **ConnectTimeout**
+  This options states the maximum number of seconds to establish a tcp 
+  connection. Default value if 10.
+ 
+  **VERBOSE**
+  Gives detailed message about the scan of all the ports. It also shows the
+  ports that were closed.
+
+## Verification Steps
+  1. Do: ```use auxiliary/scanner/portscan/tcp```
+  2. Do: ```set RHOSTS [IP]```
+  3. Do: ```set RPORT [IP]```
+  4. Do: ```run```
+  5. If any of the TCP ports were open they will be discovered , status     will   be printed indicating as such.
+
+## Scenarios
+  Example run against
+  * Metaspliotable2-Linux:192.168.45.159
```

Comment:
Change this to ```### Metasploitable 2```
since its the only host, it should be obvious so the IP isn't needed